GetToWork/src/main/webapp/login-modal.jsp

```diff
@@ -194,23 +194,72 @@ h3 {
     messagingSenderId: "70498652985"
   };
   firebase.initializeApp(config);
-  function register()
-  {
-  var db = firebase.firestore();
-  db.collection("Users").add({
-      email: document.getElementById('email').value,
-      first: document.getElementById('first').value,
-      last: document.getElementById('last').value,
-      password: document.getElementById('pass').value
-  })
-  .then(function(docRef) {
+
+var db = firebase.firestore();
+
+function validateUser()
+{
+  var email = document.getElementById('email').value;
+  var first = document.getElementById('first').value;
+  var last = document.getElementById('last').value;
+  var password = document.getElementById('pass').value;
+
+  var docRef = db.collection("Users").doc(email);
+  
+  docRef.get().then(function(doc) {
+    if (doc.exists) 
+    {
+      alert("This email address is already registered!");
+      console.log("exists");
+      return;
+    } 
+    else
+    {
+      if (email == "" || first == "" || last == "" || password == "")
```

Comment:
Maybe we could allow the message to specify which  element (email, first name, lastname, password) is incorrect in a future iteration.